docs/quickstart.md

```diff
@@ -0,0 +1,407 @@
+
+validating petstore.yaml...
+[1] petstore.yaml:15:7 at #/paths/~1pets/get/operationIds
+
+Property `operationIds` is not expected here.
+
+Did you mean: operationId ?
+
+13 | get:
+14 |   summary: List all pets
+15 |   operationIds: listPets
+16 |   tags:
+17 |     - pets
+
+Error was generated by the spec rule.
+
+
+[2] petstore.yaml:103:7 at #/components/schemas/Error/require
+
+Property `require` is not expected here.
+
+Did you mean: required ?
+
+101 |     $ref: "#/components/schemas/Pet"
+102 | Error:
+103 |   require:
+104 |     - code
+105 |     - message
+
+Error was generated by the spec rule.
+
+
+[3] petstore.yaml:65:17 at #/paths/~1pets~1{petId/get/parameters/0/name
+
+Path parameter `petId` is not used in the path `/pets/{petId`.
+
+63 |   - pets
+64 | parameters:
+65 |   - name: petId
+66 |     in: path
+67 |     required: true
+
+Error was generated by the path-parameters-defined rule.
+
+
+[4] petstore.yaml:14:7 at #/paths/~1pets/get
+
+Operation object should contain `operationId` field.
+
+12 | /pets:
+13 |   get:
+14 |     summary: List all pets
+15 |     operationIds: listPets
+ … |     < 27 more lines >
+43 |               $ref: "#/components/schemas/Error"
+44 |   post:
+45 |     summary: Create a pet
+
+Warning was generated by the operation-operationId rule.
+
+
+petstore.yaml: validated in 72ms
+
+❌ Validation failed with 3 errors and 1 warning.
+run with `--generate-ignore-file` to add all problems to ignore file.
+```
+
+Yikes! There are 3 errors!
+
+Don't worry, they were made intentionally to show the power of the tool.
+
+Let's fix the errors. Please, make the following modifications:
+
+```sh line 15
+operationIds: --> operationId:
+```
+
+```sh line 58
+/pets/{petId: --> /pets/{petId}:
+```
+
+```sh line 103
+require: --> required:
+```
+
+Once fixed, run the `lint` command again:
+
+```
+$ openapi lint petstore.yaml
+```
+
+You should get the following response:
+
+```
+$ openapi lint petstore.yaml 
+No configurations were defined in extends -- using built in recommended configuration by default.
+
+validating petstore.yaml...
+petstore.yaml: validated in 44ms
+
+Woohoo! Your OpenAPI definition is valid. 🎉
+```
+
+Congratulations! You did a great job and now your OpenAPI definition is valid!
+
+### split
+
+With this command, your OpenAPI definition file will be divided into constituent parts to follow the multi-file approach. `split` does the opposite of the [`bundle`](#bundle) command.
+
+In your terminal, type the following:
+
+```
+$ openapi split petstore.yaml --outDir openapi
+```
+
+You should get the following response:
+
+```
+$ openapi split petstore.yaml --outDir openapi
+Document: petstore.yaml is successfully split
+ and all related files are saved to the directory: openapi 
+
+petstore.yaml: split processed in 44ms
+```
+
+Cool, split completed. But what actually happened?
+
+Open the `openapi` directory. See? That's where the magic took place:
+
+```
```

Comment:
```suggestion
```bash
```

Reply:
(Not clear if we can use treeview somehow: https://prismjs.com/plugins/treeview/)